crowbar_framework/app/controllers/barclamp_controller.rb

```diff
@@ -393,14 +393,24 @@ def proposal_update
           if answer[0] == 202
             missing_nodes = answer[1].map { |node_dns| NodeObject.find_node_by_name(node_dns) }
 
-            unready_nodes = missing_nodes.map(&:alias)
+            unready_nodes = missing_nodes.select { |n| n.state != "ready" }.map(&:alias)
             unallocated_nodes = missing_nodes.reject(&:allocated?).map(&:alias)
 
-            flash[:notice] = "#{t('barclamp.proposal_show.commit_proposal_queued')}: #{(unready_nodes - unallocated_nodes).join(", ")}"
-
+            unless unready_nodes.empty?
+              flash[:notice] = "#{t('barclamp.proposal_show.commit_proposal_queued')}: #{(unready_nodes - unallocated_nodes).join(", ")}"
```

Comment:
Metrics/LineLength: Line is too long. [137/100] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)<br>Style/StringLiteralsInInterpolation: Prefer double-quoted strings inside interpolations.